src/view/MenuViewUseCaseFactory.java

```diff
@@ -0,0 +1,25 @@
+package view;
+
+import interface_adapter.ViewManagerModel;
+import interface_adapter.menu.MenuViewModel;
+import interface_adapter.statistics.StatisticsController;
+import interface_adapter.statistics.StatisticsPresenter;
+import interface_adapter.statistics.StatisticsViewModel;
+import use_case.statistics.StatisticsDataAccessInterface;
+import use_case.statistics.StatisticsInputBoundary;
+import use_case.statistics.StatisticsInteractor;
+import use_case.statistics.StatisticsOutputBoundary;
+import view.MenuView;
+
+public class MenuViewUseCaseFactory {
+    public static MenuView create(MenuViewModel menuViewModel, ViewManagerModel viewManagerModel, StatisticsViewModel
+            statisticsViewModel, StatisticsDataAccessInterface statisticsDataAccessInterface) {
+        StatisticsController statisticsController = StatisticsUseCase(viewManagerModel, statisticsViewModel, statisticsDataAccessInterface);
+        return new MenuView(menuViewModel, viewManagerModel, statisticsViewModel, statisticsController);
+    }
+    private static StatisticsController StatisticsUseCase(ViewManagerModel viewManagerModel, StatisticsViewModel statisticsViewModel, StatisticsDataAccessInterface statisticsDataAccessInterface) {
```

Comment:
```suggestion
    public static MenuView create(MenuViewModel menuViewModel, ViewManagerModel viewManagerModel, StatisticsViewModel
            statisticsViewModel, StatisticsDataAccessInterface statisticsDataAccessInterface) {
        StatisticsController statisticsController = statisticsUseCase(viewManagerModel, statisticsViewModel, statisticsDataAccessInterface);
        return new MenuView(menuViewModel, viewManagerModel, statisticsViewModel, statisticsController);
    }
    private static StatisticsController statisticsUseCase(ViewManagerModel viewManagerModel, StatisticsViewModel statisticsViewModel, StatisticsDataAccessInterface statisticsDataAccessInterface) {
```
Very minor style note, but it is a common convention to use `camelCase` for method names rather than to start with a capital letter.